package-lock.json

```diff
@@ -22289,9 +22289,9 @@
       }
     },
     "@brave/leo": {
-      "version": "git+ssh://git@github.com/brave/leo.git#26f873510cbedb9649a518a627c9e0e464540ca6",
-      "integrity": "sha512-ccw1H/GkTbpA9UuHoauxEpbVn+/6gbZPK008vnQJ330ve6Ou8Wx3417jQqXQJ+rrc5f4wr3LeoctCj1tdGL9Ig==",
-      "from": "@brave/leo@github:brave/leo#99f624799f6247f1b915e37f360028051746ead537981c7",
```

Comment:
I think somehow this has aliases our made up SHA to `26f873510cbedb9649a518a627c9e0e464540ca6` (the version installed before #29886 landed)